src/review/entities/review.entity.ts

```diff
@@ -0,0 +1,49 @@
+import { IsInt, IsString, IsOptional, Min, Max } from 'class-validator';
+import {
+  Column,
+  Entity,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+  JoinColumn,
+  CreateDateColumn,
+  Unique,
+} from 'typeorm';
+import { User } from '@app/common';
+import { Guide } from 'src/guide/entities/guide.entity';
+
+@Entity('Review')
+@Unique(['userId', 'guideId'])
+export class Review {
+  @PrimaryGeneratedColumn()
+  reviewId: number;
```

Comment:
Should be id, not review Id.

Reply:
I will fix all of the id names at once in the next pr.